shared/attributes.asciidoc

```diff
@@ -336,18 +336,19 @@ Reusable images
 //////////
 Issue and pull request URLs
 //////////
-:es-repo:    https://github.com/elastic/elasticsearch/
-:es-issue:   {es-repo}issues/
-:es-pull:    {es-repo}pull/
-:es-commit:  {es-repo}commit/
-:kib-repo:   https://github.com/elastic/kibana/
-:kib-issue:  {kib-repo}issues/
-:kib-pull:   {kib-repo}pull/
-:kib-commit: {kib-repo}commit/
-:ml-repo:    https://github.com/elastic/ml-cpp/
-:ml-issue:   {ml-repo}issues/
-:ml-pull:    {ml-repo}pull/
-:ml-commit:  {ml-repo}commit/
+:es-repo:     https://github.com/elastic/elasticsearch/
+:es-issue:    {es-repo}issues/
+:es-pull:     {es-repo}pull/
+:es-commit:   {es-repo}commit/
+:kib-repo:    https://github.com/elastic/kibana/
+:kib-issue:   {kib-repo}issues/
+:kib-pull:    {kib-repo}pull/
+:kibana-pull: {kib-repo}pull/
```

Comment:
Not mandatory, but you could make the dependency between these attributes explicit by doing this:
```suggestion
:kibana-pull: {kib-pull}
```